portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACSegments.path

```diff
@@ -99,6 +99,21 @@
     <td>//a[contains(@href, '${key_SegmentType}')]</td>
     <td></td>
 </tr>
+<tr>
+    <td>SELECT_SEGMENT</td>
+    <td>//a[@class='text-truncate' and text()='${key_segmentName}']</td>
+    <td></td>
+</tr>
+<tr>
+    <td>SELECT_SEGMENT_PROFILE</td>
+    <td>//ul[contains(@class, 'nav-root page-subnav')]//span[text()='${key_segmentProfile}']</td>
```

Comment:
hey @ViniLopes20 , try to avoid composed classes (_multiple classes with empty space between_), this way your selector will be very fragile. If you are using contains, you don't need to add multiple classes ;)